deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       serviceAccountName: grafana-operator
       containers:
         - name: grafana-operator
-          image: infoblox/grafana-operator:v3.7.0-16-gd5a70ddc
+          image: quay.io/integreatly/grafana-operator:v3.10.0
```

Comment:
My thought was that it would be easiest to maintain if we took this line unchanged from upstream.

The deploy test substitutes this line, so we're not actually using it.
https://github.com/infobloxopen/grafana-operator/blob/4d84d019de2e70e3a4bc1ae10aab2b7c406e464d/Makefile#L64

Reply:
This is a good approach. I missed this.